openmetadata-ui/src/main/resources/ui/src/pages/UserListPage/UserListPageV1.tsx

```diff
@@ -223,12 +222,13 @@ const UserListPageV1 = () => {
 
   const handleShowDeletedUserChange = (value: boolean) => {
     handlePageChange(INITIAL_PAGING_VALUE);
+    handlePageSizeChange(PAGE_SIZE_MEDIUM);
```

Comment:
Let's not mix up 2 separate fixes in a single PR. Can you please create a separate PR for the User list?